bedrock/privacy/redirects.py

```diff
@@ -0,0 +1,9 @@
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (
+    # bug 1319207
+    # 'Firefox Focus' cannot be used in de locale due to legal constraints
+    redirect(r'^de/privacy/firefox-focus/?', '/de/privacy/firefox-klar/', locale_prefix=False),
+    # special de URL should not be accessible from other locales
+    redirect(r'^(?P<locale>(?!de)\w{2,3}(?:-\w{2})?/)privacy/firefox-klar/?', 'privacy.notices.firefox-focus', locale_prefix=False),
```

Comment:
Agree using `no_redirect` with a simpler subsequent `redirect` is more readable. Will update!